bpfman/src/cli/system.rs

```diff
@@ -33,6 +33,14 @@ impl SystemSubcommand {
 }
 
 pub(crate) async fn execute_service(args: &ServiceArgs, config: &Config) -> anyhow::Result<()> {
+    initialize_bpfman().await?;
+
+    //TODO https://github.com/bpfman/bpfman/issues/881
+    serve(config, args.csi_support, args.timeout, &args.socket_path).await?;
```

Comment:
I don't think we need to pass config around like this in all the function args 

We can just have an idempotent "get config" function since it's coming from disk?